x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/find_cases.ts

```diff
@@ -117,6 +122,45 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
 
+      it('filters by severity', async () => {
```

Comment:
Could you please add an RBAC test? You can follow the test with the title `should respect the owner filter when using range queries`.

Reply:
I added this test.